AmazonChimeSDK.podspec

```diff
@@ -1,14 +1,14 @@
 Pod::Spec.new do |s|
-  s.name             = 'AmazonChimeSDK-Bitcode'
+  s.name             = 'AmazonChimeSDK'
```

Comment:
Do we have to do this? For bitcode deprecation I think we need to delete the podspec file instead.

Reply:
I renamed the podspec file, so there's no reference to bitcode.